src/components/main/view.js

```diff
@@ -105,6 +105,20 @@ const duplicateIconSvg = `<svg version="1.1" id="Copy" xmlns="http://www.w3.org/
   C18,6.448,17.553,6,17,6z M16,20h1c0.553,0,1-0.448,1-1v-1h-2V20z M16,12h2v-2h-2V12z M16,16h2v-2h-2V16z"/>
 </svg>`
 
+const mirrorIconSvg = `<svg width="21px" height="27px" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
You can compress these files with [svgo](https://github.com/svg/svgo), you can shave off > 50%.